group_vars/openstack/beegfs

```diff
@@ -28,7 +30,7 @@ beegfs_group_oss:
   flavor: "j2.small"
   image: "centos-7-20190104"
   user: "centos"
-  num_nodes: "{{ cluster_num_workers }}"
+  num_nodes: "{{ beegfs_oss_num_nodes }}"
```

Comment:
As we discussed, `cluster_num_workers` isn't part of the spec so this change isn't technically required.

It is arguably more readable with `beegfs_oss_num_nodes`.

Reply:
You need to either revert this change or change the name in `ui-meta/beegfs.yml`.